src/interfaces/INotifier.sol

```diff
@@ -33,6 +39,7 @@ interface INotifier {
     /// @param tokenId the ERC721 tokenId
     /// @param config the corresponding PositionConfig for the tokenId
     /// @param data caller-provided data that's forwarded to the subscriber contract
+    /// @dev Callers must specify a high gas limit (remaining gas should be higher than 300,000 gas) such that the subscriber can be notified
```

Comment:
```suggestion
    /// @dev Callers must specify a high gas limit (remaining gas should be higher than subscriberGasLimit) such that the subscriber can be notified
```

Reply:
it wont be 300k on all chains, plus block gas limit can change so we dont know what it will be in future